aten/src/TH/generic/THTensor.cpp

```diff
@@ -88,7 +88,15 @@ THTensor *THTensor_(newWithStorage)(THStorage *storage, ptrdiff_t storageOffset,
 THTensor *THTensor_(newWithStorage1d)(THStorage *storage, ptrdiff_t storageOffset,
                                int64_t size0, int64_t stride0)
 {
-  return THTensor_(newWithStorage)(storage, storageOffset, {size0}, {stride0});
+  THStorage *new_storage = THStorage_(new)();
+  THTensor *self = c10::make_intrusive<at::TensorImpl, at::UndefinedTensorImpl>(
+    c10::intrusive_ptr<at::StorageImpl>::reclaim(new_storage),
+    at::DispatchKey::CPUTensorId
+  ).release();
+  THTensor_(setStorageNd)(self, storage, storageOffset, 1,
```

Comment:
I'm noticing the unconditional `storage` here vs. the ternary in [newWithStorage](https://github.com/pytorch/pytorch/pull/34382/files#diff-e3bd8f02c06ae7de07fb1ace5bbd34eeL82), with AFAICT identical preceding logic... 

If that's actually an error, it seems like a consequence of the inlining, aka an argument for the way it was before. But maybe there are other motivations for inlining further up the stack?

Reply:
A: yup, answered by what's happening later in the stack

Reply:
ya sorry I didn't make that clear -- it's all handled later in the stack.  Some of this is pretty low level and can be subtle, so I wanted separate CI signal on every change.